troposphere/static/js/collections/SizeCollection.js

```diff
@@ -9,7 +9,7 @@ define(function (require) {
   return Backbone.Collection.extend({
     model: Size,
 
-    url: globals.API_V2_ROOT + "/sizes",
+    url: globals.API_V2_ROOT + "/sizes?archived=true",
```

Comment:
Hmm... ya - that's bad. 

Okay - I'll need to look for how to handle this more conditionally.